common/types.go

```diff
@@ -32,6 +32,28 @@ func ToPbECGroupElement(el *ECGroupElement) *pb.ECGroupElement {
 	return &x
 }
 
+// Pair is the same as ECGroupElement, but to be used in non EC schemes when a pair of
```

Comment:
I was thinking about this, but due to readability I decided to go for two structs. Using only Pair is an option, but let's wait and see if some other similar duplications will be considered and decide about this then.